src/seedu/addressbook/data/person/Person.java

```diff
@@ -61,6 +61,15 @@ public Address getAddress() {
         return new HashSet<>(tags);
     }
 
+    public String getPrintableString(Printable ...printables){
+        String printableString = printables[0].getPrintableString();
```

Comment:
This is correct. But to be consistent with all the instances in the list, you may create a default string and update while iterating the entire list. This will help you in future in case if you want to do something else within the loop for all the elements in the array